src/app/modules/submission/multirespondent-submission/multirespondent-submission.middleware.ts

```diff
@@ -513,8 +577,12 @@ export const validateMultirespondentSubmission = async (
                 ).map(() => undefined)
               })
               .andThen(() =>
-                // TODO: Step 4: Validate each field content with each field's validator rules individually.
-                ok(undefined),
+                validateMrfFieldResponses({
```

Comment:
Since we're potentially blocking submissions here and we don't have full confidence that we're making the right blocks. Would recommend us to implement this through a "shadow"[1] instead.

Lmk if you have similar concerns on the confidence of the validation implementation.


[1] We log that a validation has failed, instead of block. Once we have sufficient information and confidence, we can officially swap that to a block.

Reply:
done, when a validation error for v3 occurs, ValidateFieldErrorV3 will be thrown which will be logged on DD and we can use that for monitoring for the next 1-2 weeks.